Automattic/wp-calypso
# State: Implement billing receipt email sending in Redux

This PR implements the billing receipt email sending functionality in Redux. It's necessary in order to reduxify that functionality in `BillingHistoryTable`, which will help purify it. This PR implements:

* A `sendBillingReceiptEmail()` action thunk that performs a request to send the email receipt when called
* A `sendingReceiptEmail` reducer that handles the receipt email sending requests.
* A `isSendingBillingReceiptEmail` selector that we'll use to determine whether an email is being sent for a certain receipt.
* Tests for the above.

To test:

* Checkout this branch
* Verify all billing transactions test pass: 

```
npm run test-client client/state/billing-transactions/
```

* Verify tests of the new selector pass:

```
npm run test-client client/state/selectors/test/is-sending-billing-receipt-email.js 
```
